API/controllers/article-controller.js

```diff
@@ -27,3 +31,27 @@ exports.getArticles = (req, res, next) => {
     res.status(200).send({ articles });
   });
 };
+
+exports.patchArticle = (req, res, next) => {
+  const { inc_votes } = req.body;
+  const { article_id } = req.params;
+
+  if (isNaN(Number(inc_votes))) next({ status: 400, msg: "Malformed Body" });
+
```

Comment:
You don't need this logic, psql will throw this error for you and the `$` binder will protect against SQL injection